### ISSUE TITLE
Cannot read property "model" of null error when toggling extension on and off rapidly

### ISSUE THREAD

**Issue by robertknight** [hypothesis/h#3016]
I was able to reproduce the error at https://app.getsentry.com/hypothesis/prod/issues/113671701/activity/ by toggling the extension on and off rapidly on a tab open at example.com:

```
TypeError: Cannot read property 'model' of null
    at process (VM114 app.bundle.js?44d1c6:6031)
    at transformData (VM109 angular.bundle.js?dd0993:17482)
    at transformResponse (VM109 angular.bundle.js?dd0993:18258)
    at processQueue (VM109 angular.bundle.js?dd0993:22814)
    at VM109 angular.bundle.js?dd0993:22830
    at Scope.$eval (VM109 angular.bundle.js?dd0993:24082)
    at Scope.$digest (VM109 angular.bundle.js?dd0993:23898)
    at Scope.$apply (VM109 angular.bundle.js?dd0993:24190)
    at done (VM109 angular.bundle.js?dd0993:18528)
    at completeRequest (VM109 angular.bundle.js?dd0993:18726)
```

**Comment by robertknight** [comment]
Another way to reproduce this error is simply to do something which causes connections to the Hypothesis web service to fail, eg. add a bogus `/etc/hosts` entry for the domain.

**Comment by robertknight** [comment]
Fixed by 4be9ab1747fd199a2b580124c0040718f859597a
